module/userProjects/helper.js

```diff
@@ -1952,7 +1974,17 @@ module.exports = class UserProjectsHelper {
             
             let totalCount = 0;
             let data = [];
-            
+
+            let programJoined = await programUsers.findProgramIds(userId)
```

Comment:
@ankitpws Do we really need to pass programJoined flag in userAssigned function? I think this function returns user assigned projects details at  project page under assignedTome tab. when user click on a project I think userprojects/details api is called. why can't we pass this flag in that response?